PrestaShop/PrestaShop
# [+] CORE : Bring back "enable / disable on mobile" action

## Description



This pull-request brings back the mobile option on PrestaShop.
![capture du 2016-04-06 18-03-03](https://cloud.githubusercontent.com/assets/6768917/14323769/27879768-fc22-11e5-9696-c7ee46039377.png)
## Steps to Test this Fix

In order to test this pull-request:
- Go to your module catalog, tab "Installed modules" or "Notifications".
- Open the contextual menu of an **Installed** module
- Check that "Enabled mobile" or "Disable mobile" appear in the list
- Click on it and see an ajax request executed
- Everything should be went right and the other option should now appear
## Forge Ticket (optional)



This pull-request fixes the issue [BOOM-74](http://forge.prestashop.com/browse/BOOM-74)
